test/e2e/setup.go

```diff
@@ -143,7 +144,7 @@ func setup(ctx context.Context) error {
 
 	clusterName = os.Getenv("CLUSTER")
 
-	if os.Getenv("E2E_CREATE_CLUSTER") != "" {
+	if os.Getenv("CI") != "" { // always create cluster in CI
```

Comment:
```suggestion
	if os.Getenv("CI") != "" || os.Getenv("E2E_CREATE_CLUSTER") != "" {
```
??

Reply:
I think we can start dropping CREATE as we as we don't quite have usecase for it?

Reply:
We have it currently in files for backwards compatibility.

Reply:
ok